ansible/vars.yml

```diff
@@ -10,11 +10,12 @@ postgresql_release_checksum: sha256:ea2cf059a85882654b989acd07edc121833164a30340
 pgbouncer_release: "1.19.0"
 pgbouncer_release_checksum: sha256:af0b05e97d0e1fd9ad45fe00ea6d2a934c63075f67f7e2ccef2ca59e3d8ce682
 
-# to get these use `wget https://github.com/PostgREST/postgrest/releases/download/v12.0.2/postgrest-v12.0.2-ubuntu-aarch64.tar.xz -q -O- | sha1sum`
-# note: update temporarily hardcoded value over at docker/all-in-one/Dockerfile on the next PostgREST version
```

Comment:
The Dockerfile needs to be updated per this comment.

Simplest way would be to revert c318ad8ce966ef84d31fc27412f95917b7d53292.

Reply:
Done. Reverted.